src/main/java/CarRacing/CarRacingGame.java

```diff
@@ -17,16 +23,34 @@ public void gameStart() {
         gameInit();
         for(int i=0 ; i < carRacingOption.getGameCount() ; i++) {
             carList.stream().forEach( car -> {
-                car.progress();
+                car.progress(new RandomValue());
             });
             carRacingView.view(carList);
         }
+        carRacingView.resultView(gameWinner(carList));
     }
 
     private void gameInit() {
+        String[] carNames = carRacingOption.getCarNames();
         carList = new ArrayList<Car>();
         for(int i=0 ; i<carRacingOption.getCarCount() ; i++) {
-            carList.add(new Car(String.valueOf(i), new Position()));
+            carList.add(new Car(carNames[i], String.valueOf(i) , new Position()));
         }
     }
+
+    public List<Car> gameWinner(List<Car> carList) {
+        List<Car> winnersList;
+        int maxPosition = carList.stream()
+                            .max(carComparator)
+                            .get()
+                            .getLocation();
```

Comment:
`max()`는 `Optional` 을 return하여, 사용 전에 값이 존재하는지 여부를 확인해야 합니다.
지금과 같은 경우 RuntimeException이 발생할 여지를 남겨두게 돼요. 

```suggestion
                            .orElseThrow(...)
                            .getLocation();
```